### ISSUE TITLE
iframe plugin: Add option to remove the iframe from tabindex

### ISSUE THREAD

**Issue by kirkkala** [ckeditor/ckeditor4#4388]
## Type of report

Feature request

## Provide description of the new feature

Some iframed content, for example complex Microsoft Power BI visualisation can be very bad with accessibility that users keyboard-tabbing through the page content without mouse may get locked in to the iframe when represented within the content of a page.

It would come handy to have an option to add `tabindex="-1"` parameter for the iframe plugin dialog options to completely force users from getting into the iframed content when using keyboard only.

This option would enhance the aceccibility of a website by preventing users experiencing accessibility issues of iframed content embedded from sources the website administrator does not have any control to.

<img width="446" alt="iframe-remove-tabindex-option" src="https://user-images.githubusercontent.com/5946065/101274865-45cd9100-37aa-11eb-89fb-07d05dfec78a.png">

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/4389
